tests/main/snapshot-exclusions-dynamic/task.yaml

```diff
@@ -0,0 +1,124 @@
+summary: Check that the basic dynamic snapshot data exclusion functionality is ok
```

Comment:
nitpick
```suggestion
summary: Check the basic dynamic snapshot data exclusion functionality
```

Reply:
Thanks, changed this for both dynamic and static tests.